### ISSUE TITLE
Avoid breaking shipped APIs when there's really no need to? 

### ISSUE THREAD

**Issue by bgianfo** [migueldeicaza/gui.cs#1037]
Has there been any discussion as to what stability guarantees you folks are trying to target?
As a consumer if you folks could strive to avoid breaking shipped APIs when there's really no need to. 

One example I just hit, the removal of Key.ControlC (and others) in https://github.com/migueldeicaza/gui.cs/commit/01d4b4f55f68b4a7e17aebf05a2e2633f33422a0 by @BDisp needlessly breaks backwards compatibility. When the new 'improved' functionality could have lived next to the old functionality.

This is far from the only example, I've been debugging why my ListView based application displays nothing after upgrading to 0.90.3 this evening, after having to spend a bit getting it even compiling again after numerous other breaking changes.

I'm very happy folks have taken the library and run with it in terms of improvements, however it's pretty frustrating trying to track this package and take advantage of the work being done as a consumer.

**Comment by BDisp** [comment]
@bgianfo breaking the compatibility with previous versions is always annoying but is this case with the introduction of the news keys `D0 to D9` and shortcuts, was rethought about the way of representing the `Control `key combination with others keys. We concluded that the `ControlA to ControlZ` keys were duplicated with the `Control `key modifier, for e.g. `CtrlMask | ControlC` instead of `CtrlMask | C`, which is more appealing. All the breaking keys in the API was fixed and was created the `ShortcutHelper.GetModifiersKey` method to get the key with all the modifiers keys, see the `public override bool ProcessKey (KeyEvent kb)` in the `TextField`. We apologize for the inconvenience but this change was well thought out before being applied.
